services/diversity/src/util/dependency_injection_container.js

```diff
@@ -18,7 +18,7 @@ bottle.factory('logger', () => logger);
 bottle.factory('errorHandler', container => ErrorHandler(container.logger));
 bottle.factory('restAPI', () => restAPI);
 
-bottle.factory('RichnessPersistence', () => RichnessPresistence(bookshelfModels.db));
+bottle.factory('RichnessPersistence', () => RichnessPresistence(bookshelfModels.db, bookshelfModels.models, logger));
```

Comment:
I don't know the the linter doesn't catch this, but this line is over 100 characters

Reply:
Could you please add this to `.eslintrc` file (the global one), after the `"extends": "airbnb-base"` line

```
"rules": {
  "max-len": ["error", { "code": 100 }]
}
``` 

This is suppose to be one of airbnb rules...

Reply:
It was added.